odc_k8s/variables.tf

```diff
@@ -10,14 +10,21 @@ variable "owner" {
 variable "cluster_name" {
 }
 
-variable "users" {
+variable "node_role_arn" {
```

Comment:
we have: user_roles, users and node_role_arn, can these please be consistent?

Reply:
Done